.travis.yml

```diff
@@ -0,0 +1,71 @@
+git:
+  depth: 3
```

Comment:
do we need to restrict the depth? Especially with `submodules: false`

Reply:
I'm not sure how `submodules` setting is related to the `depth` setting. Could you please explain?

According to the [docs](https://docs.travis-ci.com/user/customizing-the-build#git-clone-depth) `depth` setting "truncates" the commit history up to a given number of commits. IMHO this makes sense as we don't need to clone "full" repo (50 commits in case of Travis CI). So I'd prefer to leave this setting.

Reply:
ah, my bad, didn't get it right